app/javascript/file.js

```diff
@@ -0,0 +1,15 @@
+import EmbedThis from 'src/modules/embed_this';
+import PopupPanels from 'src/modules/popup_panels';
+import { Application } from "@hotwired/stimulus"
+import TreeController from "controllers/tree_controller"
+import DownloadAllController from "controllers/download_all_controller"
+
+window.Stimulus = Application.start()
+Stimulus.register("tree", TreeController)
```

Comment:
Currently these controllers are going to be in the media bundle as well because it loads everything in `controllers` (see `controllers/index.js`).  Do we want to adjust that so these new controllers are not required for media?

Reply:
Adjusted.